requirements.txt

```
@@ -54,7 +54,7 @@ terminado==0.8.1
 testpath==0.4.2
 tornado==5.1.1
 traitlets==4.3.2
-urllib3==1.24.1
+urllib3>=1.24.2
```

maybe we can just hardcode the version? `>=` scares me because it could allow a new major version w/ breaking changes

```suggestion
urllib3==1.24.2
```

---

Oh fine if you don't want to live dangerously.